### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by ahiuchingau**
## overview

Add `CODE_OF_CONDUCT.md` for Protocol Library

## changelog

- adapted from the Contributor Covenant

## discussion

Would we like to add/or change anything?

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (Laura-Danielle) [inline CODE_OF_CONDUCT.md]: Covenant is a weird word lol

Comment (Laura-Danielle) [reply CODE_OF_CONDUCT.md]: sounds like a cult :P

Comment (Laura-Danielle) [inline CODE_OF_CONDUCT.md]: Do we want to add in things about content submitted here should only be relevant to scientific workflows or something?

Comment (kristinellis) [reply CODE_OF_CONDUCT.md]: We should probably add some language about this, I think.

Comment (kristinellis) [reply CODE_OF_CONDUCT.md]: opentrons5ever

**Pull request closed without merging**